planner/core/casetest/integration_test.go

```diff
@@ -2311,6 +2337,7 @@ func TestMppFineGrainedJoinAndAgg(t *testing.T) {
 	tk.MustExec("set @@tidb_allow_mpp=1;")
 	tk.MustExec("set @@session.tidb_broadcast_join_threshold_size = 1")
 	tk.MustExec("set @@session.tidb_broadcast_join_threshold_count = 1")
```

Comment:
Since these tests are read the input/output from json.
You can use `go test ./planner/core/... --record` to change the result quickly since I think we test these tests with late materialization as enabled will be better?

Reply:
!!! I get it, thank you!